src/test/java/core/basesyntax/service/operationhandler/impl/SubtractOperationTest.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax.service.operationhandler.impl;
+
+import core.basesyntax.bd.Storage;
+import core.basesyntax.bd.dao.StorageDao;
+import core.basesyntax.bd.dao.impl.StorageDaoImpl;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Transaction;
+import core.basesyntax.service.operationhandler.Operation;
+import core.basesyntax.service.operationhandler.OperationHandler;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class SubtractOperationTest {
+    private static StorageDao storageDao;
+    private static OperationHandler subtractOperation;
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {
+        storageDao = new StorageDaoImpl();
+        subtractOperation = new SubtractOperation(storageDao);
+    }
+
+    @Before
+    public void setUp() throws Exception {
```

Comment:
```suggestion
    public void setUp() {
```
поудаляй из всех сигнатур методов, где они не нужны